### PR TITLE
Adds subdomains attribute type

### PR DESCRIPTION
**Pull request by sudoryan**
Adds subdomains attribute type

MetadataController.getAttributeType is now async

Screen Shot 2022-12-08 at 8 54 28 AM

Will display "subdomains" attribute next to length and level attributes to show the # of subdomains issued by a domain

### CODE REVIEW

**Review by iurevych — APPROVED** [review]
👍 LGTM

File: src/models/Domain.ts [hunk 1]
```diff
@@ -129,6 +129,17 @@ export default class Domain extends Model {
...
   }
 
+  static async getSubdomainCountByParentName(
+    name: string,
+    repository: Repository<Domain> = this.getRepository(),
+  ): Promise<number> {
+    return repository.count({
+      where: { parent: { name } },
+      relations: ['parent'],
+      cache: true,
```
Comment (sammyluo) [inline src/models/Domain.ts:139]:
```suggestion
      cache: env.CACHE.IN_MEMORY_CACHE_EXPIRATION_TIME,
```
```suggestion
      cache: true,
```

**Review by sammyluo — CHANGES_REQUESTED** [review]
Looks good! 🙌  Just a small change request.

**Review by sammyluo — APPROVED** [review]
🚀 ⭐

**Pull request merged**